package.json

```diff
@@ -510,12 +510,12 @@
                 {
                     "command": "appService.appSettings.Add",
                     "when": "view == azureAppService && viewItem == applicationSettings",
-                    "group": "1_appSettingsEdit@1"
+                    "group": "1_appSettingEnv@1"
                 },
                 {
                     "command": "appService.OpenInPortal",
                     "when": "view == azureAppService && viewItem == applicationSettings",
-                    "group": "1_appSettingsEdit@2"
+                    "group": "1_appSettingsEdit@1"
```

Comment:
Should we remove "OpenInPortal" as it doesn't exist in the Functions context menu?

Reply:
Yeah, I think we may as well

Reply:
Discussed offline, but I think it's safe to remove `Open in Portal` from the list of context actions.